src/main/java/seedu/address/model/person/Person.java

```diff
@@ -153,7 +177,7 @@ private boolean equalsNullable(Object obj, Object otherObj) {
     @Override
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
-        return Objects.hash(name, phone, email, address, clientSources, note, priority);
+        return Objects.hash(name, phone, email, address, clientSources, note, isArchive, priority);
```

Comment:
Is the hash affected by the order? Since isArchive should be the last one.

Reply:
Hmm I don't think so, as long as we are consistent with the hashing
And I made isArchive come before priority in the constructor (by accident)